### DISCUSSION TITLE
How to reply to same publisher after consumed the messages from Redis streams

### DISCUSSION THREAD

**Discussion by SidathWeerasinghe** [redis/lettuce#2064] [Q&A]
I want to process the consumed message and need to reply back to the same publisher who send the message to the consumer. How can I achieve that? 

Currently, I'm using the xreadgroup and consumed the messages as below, 
` XReadArgs.StreamOffset.lastConsumed(STREAMS_KEY). `

**Comment by mp911de** [comment]
Not sure what you're trying to achieve. `Publisher` as in reactive streams is typical single-use only and you cannot replay easily messages without buffering these (via `EmitterProcessor` or similar). Other than that, this seems to be a Redis-related question.

**Reply by SidathWeerasinghe** [reply]
@mp911de 
Is there have any way to identify the publisher of the consumed message?
